staker-rewards/rewards.js

```diff
@@ -231,11 +236,15 @@ function OperatorSummary(operator, operatorParameters, operatorRewards) {
   ;(this.operator = operator),
     (this.isFraudulent = operatorParameters.isFraudulent),
     (this.factoryAuthorizedAtStart =
-      operatorParameters.authorizations.factoryAuthorizedAtStart),
+      operatorParameters.requirements.factoryAuthorizedAtStart),
     (this.poolAuthorizedAtStart =
-      operatorParameters.authorizations.poolAuthorizedAtStart),
+      operatorParameters.requirements.poolAuthorizedAtStart),
     (this.poolDeauthorizedInInterval =
-      operatorParameters.authorizations.poolDeauthorizedInInterval),
+      operatorParameters.requirements.poolDeauthorizedInInterval),
+    (this.minimumStakeAtStart =
+      operatorParameters.requirements.minimumStakeAtStart),
+    (this.wasInPoolIfRequiredAtIntervalStart =
+      operatorParameters.requirements.wasInPoolIfRequiredAtIntervalStart),
```

Comment:
Two comments:
1. For other column headers we use `*AtStart` format:`factoryAuthorizedAtStart`, `poolAuthorizedAtStart`, `minimumStakeAtStart`. Let's use it also here (`*AtIntervalStart` -> `AtStart`) to keep it consistent.
2. I think the name `wasInPoolIfRequiredAtIntervalStart` is confusing when the table is rendered. I generated the report for interval `1606928319 1607533119` and saw that operator:
    ```
     ┌─────────┬──────────────────────────────────────────────┬──────────────┬──────────────────────────┬───────────────────────┬────────────────────────────┬─────────────────────┬────────────────────────────────────┬─────────────┬─────────────────┬───────────┬────────────────┬────────────────────┬──────────────┬──────────────┬───────────┬─────────────┬───────────┬───────────┬─────────┬──────────────┬──────────────┐
     │ (index) │                   operator                   │ isFraudulent │ factoryAuthorizedAtStart │ poolAuthorizedAtStart │ poolDeauthorizedInInterval │ minimumStakeAtStart │ wasInPoolIfRequiredAtIntervalStart │ keygenCount │ keygenFailCount │ keygenSLA │ signatureCount │ signatureFailCount │ signatureSLA │  keepStaked  │ ethBonded │ ethUnbonded │ ethTotal  │ ethScore  │  boost  │ rewardWeight │ totalRewards │
     ├─────────┼──────────────────────────────────────────────┼──────────────┼──────────────────────────┼───────────────────────┼────────────────────────────┼─────────────────────┼────────────────────────────────────┼─────────────┼─────────────────┼───────────┼────────────────┼────────────────────┼──────────────┼──────────────┼───────────┼─────────────┼───────────┼───────────┼─────────┼──────────────┼──────────────┤
     (...)
     │   13    │ '0xc74D5EDAf50C15f5C64683D9CC6644121a4923f9' │    false     │          false           │         false         │           false            │        false        │                true                │      0      │        0        │   'N/A'   │       0        │         0          │    'N/A'     │    '0.00'    │  '0.00'   │   '0.00'    │  '0.00'   │  '0.00'   │ '1.00'  │    '0.00'    │    '0.00'    │
     ``` 
    My first thought after seeing `wasInPoolIfRequiredAtIntervalStart = true` was that we have a bug in the code of `staker-rewards` scripts as it is not possible that someone who has not authorized ECDSA operator contract is in a sortition pool. Then, after a while, I realized that the operator was not required to be there and that's why we have `true` in this column. However, I could realize that only because I know this code very well...

   I think we should rename this column to emphasize the logical alternative: "in pool OR not required to be in the pool". The only option I could come up with that is `inPoolOrNotRequiredInPoolAtStart` but maybe you can find a better one.

Reply:
Ad 2.
I found it very hard to figure out a name for this property. I'm affraid any name we use will require someone seeing it in the table to be familiar with rewards rules, or at least what's behind this specific condition. Maybe it's not worth finding super descriptive and complex name but use a short one that would look good in the table, e.g. `fulfilledPoolRegistrationRequirement` or `fulfilledPoolRequirement`.

The other solution is to reverse this check and align it with the condition described in Coda doc:
>* if the operator has more ETH_unbonded than the minimum unbonded value and is not in the sortition pool at T_start, they are not getting any rewards for interval T.

So the property would be named: `brokePoolRegistrationRequirement`, `failedPoolRegistrationRequirement` and it would return value oposite to the one we return currently.

Reply:
I'd add `*AtStart` no matter what because that's what we are checking. Especially that we have `poolDeauthorizedInInterval` in another column and that's the only way for the operator to leave the pool.

I'd go with one of those:
```
poolRequirementFulfilledAtStart
inPoolOrNotRequiredInPoolAtStart
```

Reply:
Oh yeah, I was so excited with the main part of the name that dropped the suffix. Of course we need `*AtStart`!